spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -247,7 +239,6 @@ def assert_table_counts
       expect(CloudNetwork.count).to                      eq network_data.networks.count
       expect(CloudSubnet.count).to                       eq network_data.subnets.count
       expect(NetworkRouter.count).to                     eq network_data.routers.count
-      expect(CloudVolume.count).to                       eq volumes_count
```

Comment:
@hsong-rh are you going to put this check back in? Right now, the tests would pass even if we don't add any volumes.